FieldOpt/Reservoir/grid/cell.h

```diff
@@ -4,9 +4,9 @@
    This file is part of the FieldOpt project.
 
    FieldOpt is free software: you can redistribute it and/or modify
-   it under the terms of the GNU General Public License as published by
-   the Free Software Foundation, either version 3 of the License, or
-   (at your option) any later version.
+   it under the terms of the GNU General Public License as published
+   by the Free Software Foundation, either version 3 of the License,
```

Comment:
Don't change the formatting of the license text without a good reason. Having the license text formatted the same way everywhere is better. The original format is copied verbatim from the [GNU GPL web page](https://www.gnu.org/licenses/gpl-howto.html).